tests/test_dms/test_dms.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import unicode_literals
+
+from botocore.exceptions import ClientError
+import boto3
+import sure  # noqa
+from moto import mock_dms
+
+
+@mock_dms
+def test_create_and_get_replication_task():
+    client = boto3.client("dms", region_name="us-east-1")
+
+    client.create_replication_task(
+        ReplicationTaskIdentifier="test",
+        SourceEndpointArn="source-endpoint-arn",
+        TargetEndpointArn="target-endpoint-arn",
+        ReplicationInstanceArn="replication-instance-arn",
+        MigrationType="full-load",
+        TableMappings='{"rules":[]}',
+    )
+
+    tasks = client.describe_replication_tasks(
+        Filters=[{"Name": "replication-task-id", "Values": ["test"]}]
+    )
+
+    tasks["ReplicationTasks"].should.have.length_of(1)
+    task = tasks["ReplicationTasks"][0]
+    task["ReplicationTaskIdentifier"].should.equal("test")
+    task["SourceEndpointArn"].should.equal("source-endpoint-arn")
+    task["TargetEndpointArn"].should.equal("target-endpoint-arn")
+    task["ReplicationInstanceArn"].should.equal("replication-instance-arn")
+    task["MigrationType"].should.equal("full-load")
+    task["Status"].should.equal("creating")
+
+
+@mock_dms
+def test_create_existing_replication_task_throws_error():
+    client = boto3.client("dms", region_name="us-east-1")
+
+    client.create_replication_task(
+        ReplicationTaskIdentifier="test",
+        SourceEndpointArn="source-endpoint-arn",
+        TargetEndpointArn="target-endpoint-arn",
+        ReplicationInstanceArn="replication-instance-arn",
+        MigrationType="full-load",
+        TableMappings='{"rules":[]}',
+    )
+
+    try:
```

Comment:
Please use the `with pytest.raises(..)` construct here - that way we don't have to have an else-statement
(Example: https://github.com/spulec/moto/blob/master/tests/test_cognitoidp/test_cognitoidp.py#L1555)